login.js

```diff
@@ -0,0 +1,65 @@
+
+function errorBoxOnOff(type, target) {
+  const targetNode = document.querySelector(`#${target}`)
+  targetNode.style.display = type === 'off' ? 'none' : 'block'
+}
+
+function focusOutCheck(event){
+  const value = event.target.value;
+  const id = event.target.id
```

Comment:
```suggestion
  const { value, id } = event.target;
```

나중에 배울 기회에 있으시겠지만 이런식으로 비구조화할당을 할 수 있습니다.